### ISSUE TITLE
BAM file per single cell or bulk?

### ISSUE THREAD

**Issue by JihedC** [yuifu/millefy#4]
Hi @yuifu,

The tool you developed seems very nice and I would like to use it for on my dataset. 
Before starting I would like to check something with you and that I didn't understand from the documentation. 

I have a BAM file for my scRNA that was generated from CellRanger or STAR solo. It contains all the reads with all barcodes. Shall I generate the bigwig file from this BAM file or shall I split it in 1 BAM file per cell?

I found another issue #3  where you mention that the user needs to split the BAM into single cell BAM using [bxtools](https://github.com/walaj/bxtools#installation). But the previous issues does not seem to be solved.

Can you let me know how I should proceed?

Best wishes,

Jihed
